cache/lru_cache.cc

```diff
@@ -463,34 +514,59 @@ Cache::Handle* LRUCacheShard::Lookup(
     const ShardedCache::CreateCallback& create_cb, Cache::Priority priority,
     bool wait, Statistics* stats) {
   LRUHandle* e = nullptr;
+  bool erase_handle_in_sec_cache{false};
```

Comment:
It might be easier to understand if it is called something like "found_dummy_entry".